src/App.tsx

```diff
@@ -1,61 +1,115 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
+  const [isErrorTitle, setErrorTitle] = useState(false);
+  const [isErrorUser, setErrorUser] = useState(false);
+  const [todos, setNewTodos] = useState(todosFromServer);
```

Comment:
```suggestion
  const [todos, setTodos] = useState(todosFromServer);
```
Set state function should be named according to the noun variable